### ISSUE TITLE
F-8 of Reg-Resp-5 error in Conformance tool on iOS

### ISSUE THREAD

**Issue by Leejigun** [fido-alliance/conformance-tool-issues#126]
## What protocol and version of the protocol are you testing?

UAF v1.0

## What is your implementation class?

Authenticator + Client Combo app in iOS.

## What is the version of the tool are you using?

v.0.8.51

## Issue description

we don't know why we failed in F-8 of Reg-Resp-5 (about Assertion does NOT contain UAuth.pub key signed by the Attestation Private Key).

Client app works without difficulty in Our FIDO Certification Server (pre-certified FIDO ® Certified Products(server)) , but we don't know why it fails with the Conformance app.

We have communicated with you a lot about debugging configuration apps in the past, so we carefully guess if this is a conformance app's bug.

Our specifications for assertion are as follows:
(It is tailored for 5.2.1 TAG_UAFV1_REG_ASSERTION in FIDO UAF Authenticator Commands.)
(https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-uaf-authnr-cmds-v1.1-id-20170202.html)
TAG_ASSERTION_INFO = 0x04,0x00,0x01,0x01,0x00,0x00,0x01}
TAG_PUB_KEY = [0x04, X (32 bytes), Y (32 bytes)] Fomat of ALG_KEY_ECC_X962_RAW 0x0100 ( in 3.6.2 Public Key Representation Formats)
(https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-registry-v1.1-id-20170202.html)

In summary, here's what we want to hear:
1). Are the inquiries we sent to you being handled?
2). If you need more time, how long will it take?
3). Check if our specifications about Assertion are okay.

**Comment by herrjemand** [comment]
@Leejigun Your encoding is correct, however it seems that your signature is invalid. Please check that your are correctly generating signature

**Comment by Leejigun** [comment]
We have a FIDO ® Certified Products(server) that is already authenticated.
Our app works well on that server without errors.
we tested it in a surrogate way, and it works well.
Please check the test APP about  F-8 of Reg-Resp-5 test

**Comment by herrjemand** [comment]
Replied privately
